service/src/main/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityRecipientsService.java

```diff
@@ -66,7 +66,11 @@ private List<String> getDesignatedLocalAuthorityContacts(CaseData caseData) {
         localAuthorityInboxes.getSharedInbox(caseData.getCaseLocalAuthority()).ifPresent(recipients::add);
 
         if (isNotEmpty(caseData.getLocalAuthorities())) {
-            final Optional<LocalAuthority> localAuthority = getDesignatedLocalAuthority(caseData);
+            Optional<LocalAuthority> localAuthority = getDesignatedLocalAuthority(caseData);
+
+            if (localAuthority.isEmpty()) {
+                localAuthority = Optional.of(caseData.getLocalAuthorities().get(0).getValue());
+            }
```

Comment:
It can be but we thought it was cleaner to put it hear instead of putting the logic inside that method with the unwrap then converting the whole thing back to an optional